pkg/metricshandler/metrics_handler.go

```diff
@@ -56,7 +56,7 @@ type MetricsHandler struct {
 }
 
 // New creates and returns a new MetricsHandler with the given options.
```

Comment:
Thanks, makes sense, we even have a comment about it :D https://github.com/kubernetes/kube-state-metrics/blob/1d61fc146160a94ad6b932b9fc5124bc43a5e00c/internal/store/builder.go#L54